tutorials/packaging/snap-a-website.md

```diff
@@ -0,0 +1,681 @@
+```
+
+Most fields are self-explanatory, but note the importance of the `Name` field. We are using "Castle Arena" here for the first time and it's how our app will be presented to users in their app list once it's installed.
+
+**We are done with packaging our app.**
+
+In the next step, we are going to run snapcraft, install our new snap and test it.
+
+## Building and testing the snap
+Duration: 1:00
+
+Now that we have created all the pieces: the app, the electron metadata (`package.json`), the snapcraft metadata (`snapcraft.yaml`) and the desktop file, it's time to run snapcraft!
+
+The command we are going to use is `snapcraft cleanbuild`: it will create an Ubuntu container using LXD, copy our project into it, download and build our part, compress the result as a snap and move the resulting snap file back into our project directory.
+
+From the root of our project (`castlearena/`), run:
+
+```bash
+snapcraft cleanbuild
+```
+
+Some details about the output you are now looking at:
+
+1. The container is created (if it's the first time you create an Ubuntu 16.04 container on this computer, an Ubuntu image is downloaded as part of the process)
+1. The project is copied in the container
+1. Snapcraft is installed in the container
+1. snapcraft pulls the source of each part
+1. ...and downloads all the required dependencies
+1. Each part is built following our instructions
+1. Snapcraft creates a staging directory where the content of our snap is curated based on instructions and parts plugin declarations
+1. The staging directory is packed into a snap
+
+When the process is over, you should see a new file at the root of your project:
+
+`castlearena_0.1_amd64.snap`
+
+Congrats, you made a snap!
+
+### Time for some testing...
+
+To install the snap on your system, run:
+
+```bash
+snap install --dangerous castlearena_0.1_amd64.snap
+```
+
+Positive
+: **What does `--dangerous` mean?**
+The `--dangerous` flag indicates that you are acknowledging that this snap could be unsafe to install, this is necessary when the snap didn't went through store security checks. Since you are the developer of the snap, you know it's safe, but the snap command doesn't and would prevent the install if you omitted the flag.
+
+Then, you can start the app with:
+
+```bash
+castlearena
+```
+
+If you followed each step, you should be greeted with a pretty fun game:
+
+![victory](https://assets.ubuntu.com/v1/10195d75-Peek+2017-11-14+17-01.gif)
+
+You should also search from it in your desktop app list and check if the icon renders nicely! If something looks wrong, at this point you probably know where to fix it: the desktop file.
+
+## What next?
+
+What a journey! If this is the first time you installed a snap or created a LXD container, you may have realized that most of the time spent in this tutorial was looking at downloads! When the tools have been used once, things get much faster, since most of them don't have to be installed or initialized anymore.
```

Comment:
realised, initialised

Reply:
I would put here the paragraph about releasing.